tests/unit/components/graphqlFormGenerator/formgenerator.vue.spec.js

```diff
@@ -166,11 +168,25 @@ const NESTED_TYPES = [
   ]
 ]
 
+const localVue = createLocalVue()
+
+Vue.use(Vuetify)
+
 describe('FormGenerator Component', () => {
-  it('should display mutation name and description', () => {
-    const localVue = createLocalVue()
-    const wrapper = mount(FormGenerator, {
+  /**
+   * @param {*} options
+   * @returns {Wrapper<FormGenerator>}
+   */
+  const mountFunction = options => {
+    const vuetify = new Vuetify()
+    return mount(FormGenerator, {
       localVue,
+      vuetify,
+      ...options
+    })
+  }
```

Comment:
Added this function as in other tests, to avoid having to `const vuetify = new Vuetify()` etc inside each test function.